configs/body_factory/default/request#uri_len_too_long

```diff
@@ -0,0 +1,15 @@
+<HTML>
```

Comment:
Now we have AUTest and Catch Unit Test framework available now. Probably good to add some tests.
In general, ATS development doesn't have any test, which is not good.